docs/design/labelling.rst

```diff
@@ -0,0 +1,800 @@
+
+Examples:
+
+**Do:** "You might need additional permissions to access the advanced settings."
+
+Do use 'might' to clarify possibility.
+
+**Avoid:** "You may need additional permissions to access the advanced settings."
+
+Avoid 'may' when 'might' will work.
+
+Inclusive language
+------------------
+
+Avoid racial, cultural, or gender bias. Ensure all words are inclusive.
+
+Pronouns
+--------
+
+Use the second person (you, your) as often as possible.
+
+Examples:
+
+**Do:** "You can update your profile settings"
+
+**Also do:** "Try refining your search criteria to find the desired results."
+
+Use the first person in headings or labels specific to the User, such as "My preferences" or "My notifications." However, switch to second person in explanatory text, like "Your notifications are updated every hour."
+
+Use the first person (we, our) to refer to the organization when appropriate, such as in requests for personal information where the user benefits from knowing why the information is needed.
+
+Example: "Why do we need your Contact details?"
+
+Active and passive voice
+------------------------
+
+The active voice is direct and emphasizes the subject of the sentence. The subject clearly "acts upon" the verb (hence, "active"). For example, "John ate the apple." In situations where either voice will work, generally choose the active voice for more directness.
+
+Examples:
+
+**Do:** Next, the User updates the profile settings.
+
+Use active voice when appropriate.
+
+**Avoid:** Next, the profile settings are updated by the User.
+
+Avoid passive voice when active voice suffices.
+
+The passive voice, however, flips the construction so the subject is secondary to the verb and object. Often, the subject is not included in the sentence. For example, "The campaign was launched by the team" or simply "The campaign was launched." Only sentences with direct objects can be constructed in passive voice, so "The team launched" cannot be passively constructed.
+
+The passive voice can create a more natural tone in certain contexts. For example, if the true subject is a system and the human is secondary, passive voice can be suitable.
+
+Examples:
+
+**Do:** The report needs to be generated.
+
+**Avoid:** Someone needs to generate the report
+
+
+Action labels
+=============
+
+Users depend on consistent labels for common actions to navigate interfaces effectively. Use this list to label actions in Mautic.
+
+A
+-
+
+Add
+^^^
+
+Takes an existing object and uses it in a new context (for example, adds an item to the cart, adds a User to a group, or adds a document to a folder).
+
+Where appropriate, combine add with the object (for example, Add User or Add role). Compare Create, Insert, New, and Upload.
+
+Apply
+^^^^^
+
+Saves changes without closing the dialog. These properties often affect subsequent system behavior.
+
+Use instead of Save changes. Compare Save and Save as.
```

Comment:
Could we use anchor links to the headings here? Makes it easier for the user to move around.

Reply:
I guess I added everywhere